packages/renderer/src/lib/images/DashboardIcon.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+export let size = '24';
```

Comment:
Nit: if size is being set in NavPage anyway, 1em is probably a better default size for reuse.

Reply:
I think it's because I don't trust default values (someone can change it) so this is why I use width like for all the other items in AppNavigation